index.js

```diff
@@ -0,0 +1,10 @@
+
+//IMPORTS
+require('dotenv').config();
+const server = require( './api/server' );
+
+//SERVER SETUP
+const PORT = 4242;
```

Comment:
you should make the PORT const have the default of process first, like this: const PORT = process.env.PORT || 4242   this way when you deploy, you dont need to refactor